setuptools/build_meta.py

```diff
@@ -209,9 +210,13 @@ def build_wheel(self, wheel_directory, config_settings=None,
                                          wheel_directory, config_settings)
 
     def build_sdist(self, sdist_directory, config_settings=None):
-        return self._build_with_temp_dir(['sdist', '--formats', 'gztar'],
-                                         '.tar.gz', sdist_directory,
-                                         config_settings)
+        orig, sdist.include_pyproject_toml = sdist.include_pyproject_toml, True
```

Comment:
This monkeypatching style is not threadsafe. Overlapping calls could leave the original unset. I don't believe thread safety is needed here.